icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2041,6 +2046,49 @@ class U_I18N_API NumberFormatterSettings {
      */
     Derived scale(const Scale &scale) &&;
 
+    /**
+     * Specifies the usage of the unit ("person", "road", "person" ...etc.)
+     *
+     * <p>
+     * NOTE: `usage` will change the output unit depending on the `Locale`  
+     * and the unit value.
+     *    For Example:
+     *        Locale: en_US
+     *        Usage : length-person
+     *        Unit  : Meter
+     *
+     *        If the unit value is 0.25, the output will be "10 inches."
+     *        If the unit value is 1.50, the output will be 
+     *                                           "4 feet and 11 inches"
+     *</p>
+     *
+     * <P>
+     * If the input usage is not exist (e.g. "dance") or is misspelled,
```

Comment:
If the input usage is not supported, such as "dance" or "pperson" (misspelling),

Reply:
I've adjusted the documentation more than was necessary, pardon me.

In the process, I dropped the "misspelling" mention - do you feel this necessary, for code? Or giving an example of unknown usages?

For a list of supported usages, I'm currently referring to CLDR's units.xml and ICU's units.txt for lack of better resources to point at at this time (maybe we should add some)...